src/ORM/DataObject.php

```diff
@@ -444,27 +520,109 @@ protected function duplicateManyManyRelations($sourceObject, $destinationObject,
     }
 
     /**
-     * Duplicates a single many_many relation from one object to another
+     * Duplicates a single many_many relation from one object to another.
```

Comment:
Point out that this doesn't copy many_many_extraFields? Needs to be mentioned in higher level docs as well

Reply:
It's on the phpdoc of cascade_duplications

```php
    /**
     * List of relations that should be cascade duplicate.
     * many_many duplications are shallow only.
     *
     * Note: If duplicating a many_many through you should refer to the
     * has_many intermediary relation instead, otherwise extra fields
     * will be omitted from the duplicated relation.
     *
     * @var array
     */
    private static $cascade_duplications = [];
 ```

Reply:
I'll add another note to the markdown docs too.

Reply:
Hm, maybe I can fix the extra fields for non-through many_many.

Reply:
No need to add the extra field duplications, but it's important to point out (as you've done in markdown)